releasewarrior/config.py

```diff
@@ -80,7 +80,9 @@ def find_repo_path(search_path):
     'cdntest',
     'mirrors',
     'pushapk',
-    'signoff',
     'publish',
+    'signoff',
```

Comment:
- We can remove `cdntest`, it's the same thing with `mirrors` and no template references it currently.x:
- We can rename `localtest` to `candidates` since it's being used in Fennec templates only and we have already switched the emails to that regards
- We need to reverse order between `signoff` and `publish`
- `wnp` stays before `mirrors`
- `beta` stays before `wnp`
- we need to add a separate step called `betasignoff` in between `beta` and `wnp`.

Following this reordering & cleanup, we need to also update *all* templates with the correct steps and order. `betasignoff` should replace existing `signoff` from release RC templates while `signoff2` should simply become `signoff`.